features/cryptocell/FEATURE_CRYPTOCELL310/trng.c

```diff
@@ -22,9 +22,10 @@
 
 #include <string.h>
 #include "trng_api.h"
+#include "mbedtls/platform.h"
 
-extern CRYS_RND_State_t     rndState;
-extern CRYS_RND_WorkBuff_t  rndWorkBuff;
+extern mbedtls_platform_context  ctx;
```

Comment:
yes. Since there isn't any `getter` function for the context, we still need access to members of it. The RNG functions need the `rndState`, which is now part of the context